### ISSUE TITLE
Device detection rely on a deprecated feature

### ISSUE THREAD

**Issue by obiot** [melonjs/melonJS#861]
see the MDN documentation : https://developer.mozilla.org/en-US/docs/Web/API/NavigatorID/userAgent#Browser_compatibility

> **Deprecated**
> This feature has been removed from the Web standards. Though some browsers may still support it, it is in the process of being dropped. Avoid using it and update existing code if possible; see [the compatibility table](https://developer.mozilla.org/en-US/docs/Web/API/NavigatorID/userAgent#Browser_compatibility) at the bottom of this page to guide your decision. Be aware that this feature may cease to work at any time.

I'm not sure since when it is the case actually, and "how long we have" but we should certainly keep an eye on it, and try to change the way we detect features it in a near future.

**Comment by parasyte** [comment]
This sums up the problem pretty well: http://webaim.org/blog/user-agent-string-history/

TL;DR: There's a cyclic conflict with attempting to distinguish between competing user agents, because competing user agents want to show off the same features as every other user agent.

**Comment by Giwayume** [comment]
That's funny, because some websites don't serve you (send a blank page or load indefinitely) if you have an empty user agent string. An example of this is deviantart.com

**Comment by obiot** [comment]
closing this one, as this not a bug in melonJS, nor bring any solution to this
